services/core-web/common/actionCreators/staticContentActionCreator.js

```diff
@@ -25,6 +25,18 @@ import {
   getVarianceDocumentCategoryOptions,
 } from "../selectors/staticContentSelectors";
 
+export const loadBulkStaticContent = () => (dispatch) => {
+  console.log("LOAD BULK STATIC CONTENT");
+  dispatch(request(reducerTypes.LOAD_ALL_STATIC_CONTENT));
+  return CustomAxios()
+    .get(ENVIRONMENT.apiUrl + API.CORE_STATIC_CONTENT, createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.LOAD_ALL_STATIC_CONTENT));
+      dispatch(staticContentActions.storeBulkStaticContent(response.data));
+    })
+    .catch(() => dispatch(error(reducerTypes.LOAD_ALL_STATIC_CONTENT)));
+};
+
```

Comment:
note that all of static content creators in this file only exist for frontend tests, we could remove all of that is we prefer.